### ISSUE TITLE
Remove duplicate error msg

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#6220]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
Search with invalid dsl: `field name not in fields created`, we need to catch the exception, and not such `"offset_iter != name_offsets_.end()" at /go/src/github.com/milvus-io/milvus/internal/core/src/common/Schema.h:141` in error msg returned
```
W
2021-06-30 04:31:42,472 - milvus_benchmark.runners.search:262 - DEBUG - {'load_time': 326.86}
2021-06-30 04:31:42,472 - milvus_benchmark.main:89 - INFO - Start run case
2021-06-30 04:31:42,472 - milvus_benchmark.runners.search:270 - DEBUG - Start run query, run 1 of 2
2021-06-30 04:31:42,999 - pymilvus.client.grpc_handler:57 - ERROR - Error: <BaseException: (code=1, message=[UnexpectedError] Assert
 "offset_iter != name_offsets_.end()" at /go/src/github.com/milvus-io/milvus/internal/core/src/common/Schema.h:141
 => Cannot find field_name: float
)>
2021-06-30 04:31:43,000 - pymilvus.client.grpc_handler:57 - ERROR - Error: <BaseException: (code=1, message=[UnexpectedError] Assert
 "offset_iter != name_offsets_.end()" at /go/src/github.com/milvus-io/milvus/internal/core/src/common/Schema.h:141
 => Cannot find field_name: float
)>
2021-06-30 04:31:43,001 - milvus_benchmark.main:99 - ERROR - Traceback (most recent call last):
  File "main.py", line 96, in run_suite
    result = runner.run_case(case_metric, **case)
  File "/src/milvus_benchmark/runners/search.py", line 272, in run_case
    _query_res = self.milvus.query(case_param["vector_query"], filter_query=case_param["filter_query"])
  File "/src/milvus_benchmark/client.py", line 46, in wrapper
    result = func(*args, **kwargs)
  File "/src/milvus_benchmark/client.py", line 289, in query
    result = self._milvus.search(tmp_collection_name, query, timeout=timeout)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 63, in handler
    raise e
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 51, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 40, in inner
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 994, in search
    return handler.search(collection_name, dsl, partition_names, fields, timeout=timeout, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 63, in handler
    raise e
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 55, in handler
    return func(self, *args, **kwargs)
```
**Steps/Code to reproduce behavior**
Follow this [guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.

**Expected behavior**

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Software conditions (OS, CPU, GPU, Memory)

- Milvus version (master or released version)

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**

**Additional context**

**Comment by czs007** [comment]
any update on this ? @FluorineDog

**Comment by FluorineDog** [comment]
fixed by #6324
